### DISCUSSION TITLE
Gyakoriságváltás

### DISCUSSION THREAD

**Discussion by mekezsolt** [nav-gov-hu/eVAT#121] [Kérdések és válaszok - Webes felület]
Üdvözlet!
Alap esetben éves ÁFA bevalló vagyok, azonban „Küszöbérték túllépés” miatt negyedévente kell bevallást beadnom.
A felületen a „Kimutatások és bevallások” fülön a gyakoriság váltásnál az okot „Küszöbérték-túllépés” -re, a gyakoriságot „Negyedévesre”, az időpontot „2024.01.01.” -re állítom és jóváhagyom, akkor kiírja, hogy „A bevallási gyakoriság módosítása sikeres!”. Viszont semmi nem változik, ugyan úgy marad az egy éves időszak (2024.01.01. - 2024.12.31).
Emiatt viszont az alábbi hiba üzenetek miatt nem tudom beadna a bevallást.

- A bevallás beérkezésének dátuma nem lehet korábbi a bevallás záró dátumánál. Érk. dát.: 20240420, Bev. zár: 20241231 (Hibakód:M044)

- Ha az éves adózó túllépi az éves gyakoriság értékhatárát a tárgyév dec. 31-ével zárt bevallásában, és az időszakban nincs közösségi adószám megállapítása (és bevallásán ehhez tartozó X jelzése), akkor a Küszöbérték túllépés rovatot jelölnie kell a Főlap (C) blokkban. (Esedékesség változik !) Egyéni vállalkozó az ingatlan sorozatjellegű értékesítése sorok levonásával határozza meg a küszöbértéket. Számítás: 2465A [01] lapokon ABS| (+36C-97C)-(+76C-98C) | >= 250 ezer Ft és/vagy 2465[01] lapokon 1B+2B+3B+4B+110B+5B+6B+7B+8B+9B+89B+90B+91B-(57B+58B) > 50.000 ezer Ft. Áfa bev. jogcím: 5, ABS((A1 36C - A1 97C) - (A1 76C - 98C C)): 1417.0 (Hibakód:M012)”

Tudnak segíteni, hogy a fenti anomáliát fel lehessen oldani?
Köszönöm!

**Comment by u319169** [comment]
Tisztelt Mekezsolt!
Az a kérdésem, hogy fennáll még a probléma? Lévén eléggé régi a bejegyzés.
Üdvözlettel: u319169

**Reply by mekezsolt** [reply]
Kedves u319169!

Többszöri egyezetés után se működik, ÁNYK-t kell használnm azóta is.
Annyi "fejlemény" törtánt, hogy már semmi nem jelenik meg az e-áfa felületemen.

Üdvözlettel:
Mihály Zsolt

**Comment by PIC18F4550** [comment]
Kedves Zsolt!
Kérem, legyen szíves a eafa_m2m@nav.gov.hu címen felvenni velem a kapcsolatot!
Köszönettel: Attila

**Comment by NTCA-JA** [comment]
Kedves Zsolt!
A javítás megtörtént és élesítésre került. Köszönjük az észrevételedet. Kérünk, légy szíves kipróbálni!
Üdvözlettel:
Informatikai HelpDesk
Nemzeti Adó- és Vámhivatal Informatikai Intézet
E-mail: [eafa_m2m@nav.gov.hu](mailto:eafa_m2m@nav.gov.hu)

**Reply by mekezsolt** [reply]
Kedves  HelpDesk!

Igen, végül sikerült megoldani.

Köszönöm szépen!

**Comment by mekezsolt** [comment]
Kedves  HelpDesk!

Igen, végül sikerült megoldani.

Köszönöm szépen!
Üdvözlettel:
Mihály Zsolt

NTCA-JA ***@***.***> ezt írta (időpont: 2026. márc. 4., Sze,
6:40):
